.github/workflows/e2e.yml

```diff
@@ -15,7 +15,7 @@ env:
   SB_CHROMIUM_PATH: /usr/bin/chromium
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref_name }}
```

Comment:
I noticed `e2e.yml` workflows were canceled for no obvious reason — this is an attempt to fix that.